src/server.ts

```diff
@@ -5,7 +5,9 @@ import { LightStreamer, LightStreamerService } from "./services/LightStreamer";
 import { logger } from "./utils";
 import { config } from "dotenv";
 
```

Comment:
We shouldn't require people to use dotenv, they can use just standard env vars if desired.

Reply:
If there's no `.env` file at the root, `dotenv` will no-op. Using a `.env` file at the root of a project is a very common pattern, and requiring that users use `DOTENV_PATH` adds more friction, IMO.

If someone doesn't want to use a `.env` file, they can just omit it.

Reply:
Cool didn't know it was noop if none existed, will revert that change.